5.0.0/configuration_parameters.md

```diff
@@ -0,0 +1,10082 @@
+used by the AC as polling interval to retrieve the updated cluster configuration
+from the API. It cannot be greater than 30m<br>
+**Options**:<br>
+**Default**: 5m<br>
+**Example**:
+
+```yaml
+sysdig:
+  secure:
+    scanning:
+      admissionControllerAPI:
+        confTTLDuration: 10m
+```
+
+## **sysdig.secure.scanning.admissionControllerAPI.pingTTLDuration**
+**Required**: `false`<br>
+**Description**: TTL of an AC ping. It should be used by the AC as polling
+interval to perform a HEAD on the ping endpoint to notify it's still alive and
+connected. It cannot be greater than 30m and it cannot be greater than
+maxDurationBeforeDisconnection<br>
+**Options**:<br>
+**Default**: 5m<br>
+**Example**:
+
+```yaml
+sysdig:
+  secure:
+    scanning:
+      admissionControllerAPI:
+        pingTTLDuration: 8m
+```
+
+## **sysdig.secure.scanning.admissionControllerAPI.clusterConfCacheMaxDuration**
+**Required**: `false`<br>
+**Description**: Max duration of the cluster configuration cache. The API returns
+this value as max-age in seconds and the FE uses it for caching the cluster
+configuration. FE also asks for a new cluster configuration using this value
+as time interval. It cannot be greater than 30m<br>
+**Options**:<br>
+**Default**: 5m<br>
+**Example**:
+
+```yaml
+sysdig:
+  secure:
+    scanning:
+      admissionControllerAPI:
+        clusterConfCacheMaxDuration: 9m
+```
+
+## **sysdig.scanningAnalysiscollectorConcurrentUploads**
+**Required**: `false`<br>
+**Description**: Number of concurrent uploads for Scanning Analysis Collector<br>
+**Options**:<br>
+**Default**: "5"<br>
+**Example**:
+
+```yaml
+sysdig:
+  scanningAnalysiscollectorConcurrentUploads: 5
+```
+
+## **sysdig.scanningAlertMgrForceAutoScan**
+**Required**: `false`<br>
+**Description**: Enable the runtime image autoscan feature. Note that for adopting a more distributed way of scanning runtime images, the Node Image Analyzer (NIA) is preferable.<br>
+**Options**:<br>
+**Default**: `false`<br>
+**Example**:
+
+```yaml
+sysdig:
+  scanningAlertMgrForceAutoScan: false
+```
+
+## **sysdig.secure.scanning.veJanitor.cronjob**
+**Required**: `false`<br>
+**Description**: Cronjob schedule<br>
+**Options**:<br>
+**Default**: "0 0 * * *"<br>
+**Example**:
+
+```yaml
+sysdig:
+  secure:
+    veJanitor:
+      cronjob: "5 0 * * *"
+```
+
+## **sysdig.secure.scanning.veJanitor.anchoreDBsslmode**
+**Required**: `false`<br>
+**Description**: Anchore db ssl mode. More info: https://www.postgresql.org/docs/9.1/libpq-ssl.html<br>
+**Options**:<br>
+**Default**: "disable"<br>
+**Example**:
+
+```yaml
+sysdig:
+  secure:
+    veJanitor:
+      anchoreDBsslmode: "disable"
+```
+
+## **sysdig.secure.scanning.veJanitor.scanningDbEngine**
+**Required**: `false`<br>
+**Description**: which scanning database engine to use. <br>
+**Options**: mysql<br>
+**Default**: "mysql"<br>
+**Example**:
+
+```yaml
+sysdig:
+  secure:
+    veJanitor:
+      scanningDbEngine: "mysql"
+```
```

Comment:
Does this still make sense?

Reply:
same as above

Reply:
I need to check the code to find the answer

Reply:
this is now changed:
https://github.com/draios/installer/blob/onprem-5-0-0/installer/pkg/installer/defaults.go#L428

updating accordingly